src/asha_bt.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <cstdint>
+#include "btstack.h"
+
+#include "hearing_aid.hpp"
+
+namespace asha {
+
+enum class ScanState {
+    Stop,
+    Scan,
+    IdentityResolving,
+    Connecting,
+    Disconnecting,
+    ServiceDiscovery,
+    CharDiscovery,
+    ReadROP,
+    ReadPSM,
+    SubscribeASPNotification,
+    Finalizing,
+    Complete,
+};
+
+constexpr uint16_t buff_size_sdu = 161;
```

Comment:
This needs to be at least 167, to account for the l2cap header

Reply:
As far as I'm aware, the SDU is the application data size, and does not contain l2cap headers. That's the PDU, and I believe btstack handles that automatically.